metadata-ingestion/src/datahub/ingestion/source/unity/source.py

```diff
@@ -179,6 +188,9 @@ def __init__(self, ctx: PipelineContext, config: UnityCatalogSourceConfig):
         self.table_refs: Set[TableReference] = set()
         self.view_refs: Set[TableReference] = set()
         self.notebooks: FileBackedDict[Notebook] = FileBackedDict()
+        self.view_definitions: FileBackedDict[
+            tuple[TableReference, str]
```

Comment:
looks like there's a lint issue on python 3.7
```suggestion
            Tuple[TableReference, str]
```

Reply:
done